src/actions/global/globalactions/EditAnnotationSizeGlobalActionTest.java

```diff
@@ -0,0 +1,40 @@
+package actions.global.globalactions;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import paintcomponents.annotations.TextAnnotation;
+import paintcomponents.data.DataDisplayPaintComponent;
+import ui.PaintPanel;
+import actions.global.ActionName;
+
+public class EditAnnotationSizeGlobalActionTest {
+
+	@SuppressWarnings("deprecation")
```

Comment:
There should not be deprecation surpasses. Find out what is the modern way of doing asserts. "Google Java Junit 4 test example"